dev/az/gcloud-sqlproxy.yaml

```diff
@@ -0,0 +1,19 @@
+---
+enabled: true
+googleServiceAccount: az-proxy-sa@broad-jade-dev.iam.gserviceaccount.com
+cloudsql:
+  instances:
+    # GCP instance name.
+    - instance: "jade-postgres-11-8a00fd4d3b"
+      # GCP project where the instance exists.
+      project: "broad-jade-dev"
+      # GCP region where the instance exists.
+      region: "us-central1"
+      # Port number for the proxy to expose for this instance.
+      port: 5432
+rbac:
+  create: true
+networkPolicy:
+  enabled: false
+nodeSelector:
+  cloud.google.com/gke-nodepool: dev-node
```

Comment:
We no longer want to specify the node selector - https://github.com/broadinstitute/datarepo-helm-definitions/commit/cc375312d1b15e1972786c915ae0d6d1931f2455#diff-75712d79bca7d2a8018223e71993a58e538fb59f5a0f7473be8819da74e0d075